src/components/styles/Book.module.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500&display=swap');
+
+* {
+  font-family: 'Montserrat', sans-serif;
+}
+
+:root {
+  --color-primary: #0290ff;
+  --color-primary-dark: #4386bf;
+  --color-primary-white: #fff;
+  --color-primary-dark-white: #e8e8e8;
+}
+
+.container {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But as said by the previous reviewer, you can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

Reply:
thanks a lot, @moise10r for your review I have a question I can make a single flex property and imported it to all the CSS files?

Reply:
Hi @Nicolaswg,
Yeah, you can do it only if you use CSS `variable ` like this :point_down: 

- CSS
```
:root {
  --primary: #d3d3d3;
  --secondary: #ec5242;
  --text-color: #272a31;
  --background: rgba(230, 230, 230, 0.479);
  --weight: bold;
}



header nav.backg {
  height: 80px;
  background-color: var(--primary);
}

nav .nav_menu li:last-child {
  padding: 4px 14px;
  color: var(--secondary);
  border: 5px solid var(--secondary);
}
```